src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,171 @@
 package core.basesyntax;
-
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
```

Comment:
The class `Node<T>` should be an inner class of `MyLinkedList<T>`, not a static nested class.

Reply:
Fix the comment above, also innere classes should be placed at the end of outer classes